src/core/Chip/Chip.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { Component, ReactNode } from 'react';
+import classnames from 'classnames';
+import { default as styled } from 'styled-components';
+import { baseStyles } from './Chip.baseStyles';
+import { HtmlButton, HtmlSpan } from '../../reset';
+import { TokensProp, InternalTokensProp } from 'core/theme';
+import { withSuomifiDefaultProps } from '../theme/utils';
+import { Icon } from '../Icon/Icon';
+import { logger } from '../../utils/logger';
+import { VisuallyHidden } from '../../components/Visually-hidden/Visually-hidden';
+
+const baseClassName = 'fi-chip';
```

Comment:
We had this classnames object somewhere where you could just use e.g. classnames.disabled in the code. I personally preferred that over this more traditional solution.

Reply:
I think I used that in Checkbox. Didn't remember which one we agreed on. I'll implement it here as well.

Reply:
now changed.